.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -27,6 +27,12 @@ fun shouldBeSkipped(subProject: GradleSubproject, testConfig: TestCoverage): Boo
     return testConfig.os.ignoredSubprojects.contains(subProject.name)
 }
 
+val cleanDaemonsBeforePerformanceTest = """
+    free -m
+    ps aux | grep GradleDaemon | grep -v grep | awk '{print ${'$'}2}' | xargs kill -9
```

Comment:
* We may want to kill the Gradle workers as well
* would it make sense to call `--stop` first?

Reply:
I thought the worker daemons are forked by Gradle daemons, so killing Gradle daemons will also terminate worker daemons, right? @big-guy @ghale 

> would it make sense to call --stop first?

It probably makes sense, but I don't think that's necessary. When executing this script, the build agent is refreshed to start a new build, so it's safe to kill all previous daemons (and clear all its data).

Reply:
>  so killing Gradle daemons will also terminate worker daemons, right?

I asked them the same thing on Slack, the answer was: eventually. That's what I saw as well locally, and here we need to kill them now, eventually would not be okay, right?